### PR TITLE
Blood Content Alcohol

### LINKED ISSUE #1018: Blood alcohol content
# Aim

-  this script to know how much drunk you are!! 

# Details

-  it tell you how amount of alcohol you drink and what is percentage of alcohol content in the blood

### PR DESCRIPTION
**Pull request by soumik2012**
# Description

I have added the folder Blood Content Alcohol in which all the necessary file are there. 

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [X] Yes

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

## Project Metadata

`` If there is no-file/nothing to fill the below fields with, then type: none ``

`` Example: `` If no requirements.txt needed/present, then type none in ``Requirments``.

Category:
- [x] Calculators
- [ ] AI/ML
- [ ] Scrappers
- [ ] Social_Media
- [ ] PDF
- [ ] Image_Processing
- [ ] Video_Processing
- [ ] Games
- [ ] Networking
- [ ] OS_Utilities
- [ ] Automation
- [ ] Cryptography
- [ ] Computer_Vision
- [ ] Fun
- [ ] Others

Title: Blood Content Alcohol

Folder: Blood Content Alcohol

Requirements: none

Script: Blood_Alcohol_Content.py

Arguments: none

Contributor: soumik2012 

Description: Want to know how much amount of alcohol you drank? Well, the Blood alcohol content program will help you find out the alcohol intake.

### CODE REVIEW

**Comment by soumik2012** [comment]
@kaustubhgupta  please check my PR

**Comment by soumik2012** [comment]
@kaustubhgupta  please check my PR

File: Blood Content Alcohal/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Blood Alcohol Content
+
+Want to know how much amount in alcohol drank a man/woman was? well Blood alcohol content program will help you find out how much  alcohol  he/her was drank.
```
[the remark this answers is not in the document]
Comment (soumik2012) [reply Blood Content Alcohal/README.md]: @kaustubhgupta  please check my PR I have done modification

**Review by kaustubhgupta — APPROVED** [review]

**Review by santushtisharma10 — APPROVED** [review]

**Comment by soumik2012** [comment]
@antrikshmisri, please review my PR

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**